src/api/procedures/__tests__/cancelProposal.ts

```diff
@@ -0,0 +1,94 @@
+import sinon from 'sinon';
+
+import { isAuthorized, Params, prepareCancelProposal } from '~/api/procedures/cancelProposal';
+import { PostTransactionValue } from '~/base';
+import { Context } from '~/context';
+import { AccountKey } from '~/polkadot';
+import { dsMockUtils, entityMockUtils, procedureMockUtils } from '~/testUtils/mocks';
+import { Mocked } from '~/testUtils/types';
+import { PolymeshTx } from '~/types/internal';
+import * as utilsModule from '~/utils';
+
+jest.mock('~/api/procedures/utils');
+
+describe('cancelProposal procedure', () => {
```

Comment:
Add a test that verifies that the assert function is called with the right arguments